src/dex/generic-rfq/rate-fetcher.ts

```diff
@@ -425,6 +427,7 @@ export class RateFetcher {
         this.dexHelper.config.data.network,
```

Comment:
I think it makes sense to add `takerAddress: addressSchema.required(),` to `firmRateResponseValidator.order` validator

Reply:
Actually it was already there but I missed [this validation rule](https://github.com/paraswap/paraswap-dex-lib/blob/master/src/dex/generic-rfq/validators.ts#L171) which enforced augustus addresss. For now it's ok but we'll have to relax it for the future, doing it now